packages/docusaurus-utils/src/index.ts

```diff
@@ -51,7 +51,6 @@ export {
   parseURLPath,
   serializeURLPath,
   addLeadingSlash,
-  addTrailingSlash,
   removeTrailingSlash,
```

Comment:
We probably want to move all those 3 trailingSlash methods to the common package for consistency